### DISCUSSION TITLE
Transfer thumbnail definitions

### DISCUSSION THREAD

**Discussion by kopieMK** [pimcore/pimcore#15464] [Q&A / Support] [answered]
Hello. 

Is it possible to export or transfer the thumbnail definitions defined in Pimcore to a second instance of Pimcore, or in any other way? In my case, I want to transfer them from my testing system to the live system. I am using Pimcore 10.5.*

 I found a file called image-thumbnails.php. Can I simply upload this file to my new instance?

Best regards
Magnus

**Comment by aryaantony92** [comment, marked as answer]
You can just copy the existing file 'image-thumbnails.php' to the new instance and then run the command 'pimcore:thumbnails:image' to pre-generate the thumbnails. You can also refer this link for more deployment scenarios https://pimcore.com/docs/pimcore/10.5/Development_Documentation/Deployment/Deployment_Tools.html#page_Pimcore-Configurations

**Reply by kopieMK** [reply]
Thank you @aryaantony92 

This solution works for me. 

Have a nice day!

**Reply by aryaantony92** [reply]
@kopieMK Thanks for the update. Wish you too a nice day!
